### PR TITLE
Video homepage

### PR DESCRIPTION
**Pull request by LizBaker**

### CODE REVIEW

**Comment by clarkmcadoo** [comment]
getting a tiiiiiiny bit of side scroll on mobile:
Screen Shot 2023-02-27 at 8 46 11 AM

**Comment by clarkmcadoo** [comment]
code-wise, everything looks good! 👍

File: src/components/HomepageVideo.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <PageContainer>
+        <WelcomeContainer>
+          <h1>{t('homepageVideo.header')}</h1>
+          <p>{t('homepageVideo.p1')}</p>
+          <p>{t('homepageVideo.p2')}</p>
+          <SearchInput
```
Comment (tabathadelane) [inline src/components/HomepageVideo.js:24]: Can we add a unique tessen value to this?

**Review by tabathadelane — CHANGES_REQUESTED** [review]

File: src/components/HomepageVideo.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      flex-direction: column;
+      padding-top: 2rem;
+      .dark-mode & {
+        background-image: none;
+      }
+    }
+  `;
+  const WelcomeContainer = styled.div`
+    grid-area: welcome;
+    padding-right: 3rem;
```
Comment (clarkmcadoo) [inline src/components/HomepageVideo.js:52]: can we get rid of this on mobile? it'll just make the search bar and video look even when they're stacked
Screen Shot 2023-02-27 at 3 25 49 PM
vs:
Screen Shot 2023-02-27 at 3 25 58 PM

Comment (LizBaker) [reply src/components/HomepageVideo.js]: tis gone!

**Review by tabathadelane — APPROVED** [review]

**Review by clarkmcadoo — DISMISSED** [review]

**Review by clarkmcadoo — APPROVED** [review]

**Pull request merged**